api/Navigator.json

```diff
@@ -1735,11 +1735,22 @@
             "chrome_android": {
               "version_added": "40"
             },
-            "edge": {
-              "version_added": "17"
-            },
+            "edge": [
+              {
+                "version_added": "17"
+              },
+              {
+                "version_added": "16",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Enable service workers"
+                  }
+                ]
+              }
+            ],
             "edge_mobile": {
-              "version_added": "17"
+              "version_added": null
```

Comment:
See https://github.com/mdn/browser-compat-data/pull/1881#discussion_r183205666 for reasoning.